### PR TITLE
added new types for manual corrections and api for that

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

**Comment by hasmikharutyunyan** [comment]
Methods are very similar and similar with existing methods, maybe needed think about mere them

File: Api/AdministratorServices/ICounterpartyAccountService.cs [hunk 1]
```diff
@@ -16,5 +16,9 @@ public interface ICounterpartyAccountService
         Task<Result> SubtractMoney(int counterpartyAccountId, PaymentCancellationData data, UserInfo user);
 
         Task<Result> TransferToDefaultAgency(int counterpartyAccountId, MoneyAmount amount, UserInfo user);
+
+        Task<Result> ManualAdd(int counterpartyAccountId, PaymentData data, UserInfo user);
```
[the remark this answers is not in the document]
Comment (hasmikharutyunyan) [reply Api/AdministratorServices/ICounterpartyAccountService.cs:20]: reasons are differenet, that is why I have written that maybe we can combine those methods together

File: Api/AdministratorServices/IAgencyAccountService.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using HappyTravel.Edo.Api.Models.Payments;
+using HappyTravel.Edo.Api.Models.Users;
+
+namespace HappyTravel.Edo.Api.AdministratorServices
+{
+    public interface IAgencyAccountService
+    {
+        Task<Result> ManualAdd(int agencyAccountId, PaymentData paymentData, UserInfo userInfo);
```
[the remark this answers is not in the document]
Comment (kirillta) [reply Api/AdministratorServices/IAgencyAccountService.cs:10]: Or AddManual

**Comment by kirillta** [comment]
So what's up with this PR?

**Comment by hasmikharutyunyan** [comment]
> So what's up with this PR?

I am waiting for review

File: HappyTravel.Edo.UnitTests/Tests/AdministratorServices/CounterpartyAccountServiceTests/ManualCorrectionTests.cs [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+            var (isSuccess, _, error) = await _counterpartyAccountService.SubtractManually(
+                1, new PaymentData(1, Currencies.USD, "not empty reason"), _user);
+
+            Assert.True(isSuccess);
+            Assert.Equal(999, affectedAccount.Balance);
+        }
+
+
+        private void SetupInitialData()
```
Comment (kirillta) [inline HappyTravel.Edo.UnitTests/Tests/AdministratorServices/CounterpartyAccountServiceTests/ManualCorrectionTests.cs:138]: You should use class fixtures for that

**Review by kirillta — APPROVED** [review]

**Pull request merged**